release-notes/opensearch.release-notes-2.6.0.md

```diff
@@ -0,0 +1,57 @@
+## 2023-02-22 Version 2.6.0 Release Notes
+
+## [2.6]
+### Added
+- Add index create block when all nodes have breached high disk watermark ([#5852](https://github.com/opensearch-project/OpenSearch/pull/5852))
+- Add cluster manager throttling stats in nodes/stats API ([#5790](https://github.com/opensearch-project/OpenSearch/pull/5790))
+- Add support for feature flags in opensearch.yml ([#4959](https://github.com/opensearch-project/OpenSearch/pull/4959))
+- Add query for initialized extensions ([#5658](https://github.com/opensearch-project/OpenSearch/pull/5658))
```

Comment:
We should remove this as it's part of extensibility.

Reply:
For any updates of the release notes we should start with a PR on main and then backport them to the relevant branches.

Reply:
@owaiskazi19 Feel free to make a PR on main for this and any other edits

Reply:
As per @andrross 's comment, i'll create a PR to `main` to remove these. Are these the only 4 lines to remove?

Reply:
@owaiskazi19 please go ahead and create a PR for these changes, or let me know if you'd like me to

Reply:
I was referring to just this one line.

Reply:
PR for the same: https://github.com/opensearch-project/OpenSearch/pull/6451